### ISSUE TITLE
Failed to run stereo using Covins

### ISSUE THREAD

**Issue by clean-shero** [VIS4ROB-lab/covins#55]
I used the vins-fusion front-end and was unable to run the Euroc Stereo dataset successfully. The following error occurred. Is covins only applicable to data with imu?
![image](https://github.com/VIS4ROB-lab/covins/assets/40481117/db93e31e-3c46-425c-8149-afa0c036d1ca)

**Comment by manthan99** [comment]
Hi
You will need to make modifications to the VINS codebase in order to use the Stereo with COVINS back-end. An alternative would be to use Front-end Wrapper along with the VINS stereo Front-end and COVINS-G back-end. I would recommend checking this out in order to use the front-end wrapper.

https://github.com/VIS4ROB-lab/covins/blob/master/docs/run_COVINS-G.md#ros-based-front-end-wrapper

**Comment by clean-shero** [comment]
YES, I used VINS-COVINS-adaptation and successfully ran the Euroc dataset with imu. However, the system seems to require imu by default, and the above error occurred. My purpose is to use euroc_stereo_config.yaml

**Comment by clean-shero** [comment]
> YES, I used VINS-COVINS-adaptation and successfully ran the Euroc dataset with imu. However, the system seems to require imu by default, and the above error occurred. My purpose is to use euroc_stereo_config.yaml

YES, I used VINS-COVINS-adaptation and successfully ran the Euroc dataset with imu. However, the system seems to require imu by default, and the above error occurred. My purpose is to use euroc_stereo_config.yaml
